README.md

```diff
@@ -41,4 +60,34 @@ https://stackoverflow.com/questions/72357737/i-am-getting-this-error-failed-to-s
 PR link
 ```
 https://github.com/kovaku/spring-boot-user-admin-rest-service/pull/1
-```
\ No newline at end of file
+```
+
+Dockerize the webservice & build image during mvn build :
```

Comment:
Nice trick to collect these as reference, I did the same when I worked on my pet project